packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -145,9 +146,11 @@ class AppContext(
     get() = legacyModule()
 
   /**
-   * Provides access to the react application context
+   * Provides access to the [Context] (it's [ReactApplicationContext] under the hood, but we don't
+   * want to expose this type via this module's public API).
+   * @see [ContextProvider] - this interface is not implemented here as it forces non-null return type
    */
-  val reactContext: Context?
+  val context: Context?
```

Comment:
I wonder if we should change this name because `reactContext` is more specific about what type of context this actually is. However, in the pure Android world, we have `Context` meaning activity context or application context and it's not distinguishable by name unless explicitly called `Context.getApplicationContext()`.